src/app/list-item/list-item.component.ts

```diff
@@ -0,0 +1,15 @@
+import { Component } from '@angular/core';
+import { DataService } from '../service/data-service.service';
+
+@Component({
+  selector: 'app-list-item',
```

Comment:
`list-item` is a bad name for a component that manages lists, `list` is more than enough